cura/Arranging/Nest2DArrange.py

```diff
@@ -108,20 +107,25 @@ def findNodePlacement(self) -> Tuple[bool, List[Item]]:
                 item = Item(converted_points)
                 item.markAsFixedInBin(0)
                 node_items.append(item)
-                num_disallowed_areas_added += 1
 
-        config = NfpConfig()
-        config.accuracy = 1.0
-        config.alignment = NfpConfig.Alignment.DONT_ALIGN
-        if self._lock_rotation:
-            config.rotations = [0.0]
+        tried_strategies = [NfpConfig.Alignment.CENTER] * 3 + [NfpConfig.Alignment.BOTTOM_LEFT] * 3
+        found_solution_for_all = False
+        while not found_solution_for_all and len(tried_strategies) > 0:
```

Comment:
I'm not sure about the naming of the variable `tried_strategies`. Since it is past-tense it implies the stragegies are already performed (and rejected since we continued). I would rename the variable to `strategies` instead. 

Additionally, I'm not a big fan of the `while ... len(stragegies) > 0` and `tried_strategies = tried_strategies[1:]`. This is just a more "complicated" loop in my opinion. The `while not found_solution_for_all` is nice though. However, I would instead write something like
```suggestion
        for strategy in strategies
            ....
            if found_solution_for_all:
               break
```
But that is more personal taste

Reply:
I agree with the improper variable naming, and will change that.
However, about the change of the loop, this is the recommended way. Some purists would say that you should never use a break/continue statement, because they interrupt the flow execution and allow for a "jump", like a "goto" :smile: . I do use them sometimes, but in this case I find it more readable because you can know when the loop will stop without having to look into it.